docs(instances): fix cloud-init on debian 12 (#1854)
scaleway/docs-content: compute/instances/troubleshooting/fix-cloud-init-debian12.mdx

---
meta:
  title: Fix the wrong user configuration of cloud-init on Debian 12 (Bookworm)

>   description: This page helps you to fix the wrong user configuration of cloud-init on Debian 12 (Bookworm)

Reviewer: ```suggestion
  description: This page helps you fix the wrong user configuration of cloud-init on Debian 12 (Bookworm)
```

content:
  h1: Fix the wrong user configuration of cloud-init on Debian 12 (Bookworm)
  paragraph: This page helps you fix the wrong user configuration of cloud-init on Debian 12 (Bookworm)
tags: debian12 debian bookworm cloud-init
dates:
  validation: 2023-06-16
  posted: 2023-05-04
categories:
  - compute
---

> Debian 12 (Bookworm) Instances created before June 2nd, 2023 were delivered with a version of `cloud-init` preinstalled that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />

Author: ```suggestion
Debian 12 (Bookworm) Instances created before June 2nd, 2023 were delivered with a preinstalled version of `cloud-init` that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />
```

> Debian 12 (Bookworm) Instances created prior to 2023-06-22 were delivered with a version of `cloud-init` preinstalled that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />

Reviewer: ```suggestion
Debian 12 (Bookworm) Instances created prior to 2023-06-22 were delivered with a preinstalled version of `cloud-init` that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />
```

> Debian 12 (Bookworm) Instances created prior to 2023-06-22 were delivered with a version of `cloud-init` preinstalled that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />

Reviewer: ```suggestion
Debian 12 (Bookworm) Instances created before June 2nd, 2023 were delivered with a version of `cloud-init` preinstalled that came with a wrong user configuration made by the custom-built upstream `cloud-init 23.2`.<br />
```

> The issue has been fixed for Instances created past the above mentioned date. The Debian 12 image now uses the official Debian Unstable `cloud-init` package.<br />

Reviewer: ```suggestion
The issue has been fixed for Instances created  after June 2nd, 2023. The Debian 12 image now uses the official Debian Unstable `cloud-init` package.<br />
```
